frame/dvm/src/account_basic.rs

```diff
@@ -16,6 +16,8 @@
 // You should have received a copy of the GNU General Public License
 // along with Darwinia. If not, see <https://www.gnu.org/licenses/>.
 
+//! The module that manages the basic account info display in dvm.
```

Comment:
pallet

Reply:
`account_basic` is just a mod, not a pallet.

Reply:
How about rust module

rust/runtime module make people confuse